### ISSUE TITLE
Invalid reformating when "namespace" is used to point to current namespace

### ISSUE THREAD

**Issue by pmartin** [FriendsOfPHP/PHP-CS-Fixer#1514]
I've found a situation in which php-cs-fixer 1.10.2 inserts newlines before the `namespace` keyword but should not.

Minimalist PHP file to reproduce :

``` php
<?php
namespace EspaceDeNoms;

class MaClasse
{

}

var_dump(namespace\MaClasse::class);

$obj = new namespace\MaClasse();
var_dump($obj);
```

This is valid PHP code (even if `namespace` is not often used this way), executing it leads to the following output :

```
$ php bug-php-cs-fixer-keyword-namespace.php
string(21) "EspaceDeNoms\MaClasse"
object(EspaceDeNoms\MaClasse)#1 (0) {
}
```

Trying to reformat it with `php-cs-fixer fix bug-php-cs-fixer-keyword-namespace.php` gives the following result :

``` php
<?php

namespace EspaceDeNoms;

class MaClasse
{
}

var_dump(

namespace\MaClasse::class);

$obj = new

namespace\MaClasse();

var_dump($obj);
```

Newlines are added before each `namespace`.

OK for newlines when `namespace` is used to declare a namespace.
But there should not be newlines inserted when `namespace` is used to designate the current namespace.

**Comment by keradus** [comment]
I just love when PHP use same token for different meaning :/

**Comment by keradus** [comment]
looks like we could use next Transformer

**Comment by gharlan** [comment]
> looks like we could use next Transformer

here it is: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Tokenizer/Transformer/NamespaceOperatorTransformer.php ;)

@pmartin it is fixes on 2.x

**Comment by keradus** [comment]
hahahahaha :D

**Comment by SpacePossum** [comment]
@pmartin just checked again, this is indeed fix on master and on v2.0.0 (which has been released now :) )

reminder to myself to write a fix on the 1.x line

**Comment by keradus** [comment]
Closing as fixed.
@SpacePossum don't backport it.
